tests/common/playground.rs

```diff
@@ -20,6 +20,7 @@ pub struct PerTestOptions {
     no_default_collection: bool,
 }
 
+#[allow(dead_code)]
```

Comment:
I think it's because the method `set_no_default_collection` is only used in tests, so the compiler thinks that it isn't used. We could get around this by creating a test mod and putting this in in but that gets a bit messier so it's easier to just allow it imo.